xfce-extra/xfce4-kbdleds-plugin/xfce4-kbdleds-plugin-0.2.3.ebuild

```diff
@@ -0,0 +1,43 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit xdg-utils
+
+DESCRIPTION="A panel plug-in to show state of Caps, Num and Scroll Lock keys"
+HOMEPAGE="https://github.com/oco2000/xfce4-kbdleds-plugin"
+SRC_URI="https://github.com/oco2000/${PN}/archive/${P}.tar.gz"
+S="${WORKDIR}/${PN}-${P}"
+PATCHES=( "${FILESDIR}/fix-build-0.2.3.patch" )
```

Comment:
Please move it below all other vars, into a separate group. Also, please start the patch name with `${P}-`.

Reply:
Moved and file renamed.